documentation/guides/api/develop.md

```diff
@@ -0,0 +1,3 @@
+# Development guide
+
+Here's the comprehensive guide for developing the Openverse API.
```

Comment:
Did we have any info from elsewhere we could pull into this doc? Or will this be expanded on later? I think I might prefer _not_ having a placeholder page to prevent giving the false impression we have docs about something 😅

Reply:
Good point, I dropped the stubs from the table of contents in 3972320. That should prevent them from showing up until we've populated them.

Reply:
Perfect!